invenio_app_rdm/config.py

```diff
@@ -123,6 +124,12 @@
 from invenio_vocabularies.contrib.names.datastreams import (
     VOCABULARIES_DATASTREAM_WRITERS as NAMES_WRITERS,
 )
+from invenio_vocabularies.contrib.subjects.datastreams import (
```

Comment:
[Subjects also has readers defined](https://github.com/inveniosoftware/invenio-vocabularies/blob/master/invenio_vocabularies/contrib/subjects/datastreams.py#L33-L34), so I think they should be added here too.